docs/users_guide.md

```diff
@@ -243,6 +243,40 @@ For both hooks, the environment variable `CUSTOM_DIR` refers to the directory co
           RECIPE_FILE \
           COLLECTION_ID
 
+#### Koji build
+
+1.  Ensure that you have configuration for the desired koji instance
+    (i.e.  Fedora Koji, CentOS CBS, …) installed on your system.
```

Comment:
I want links for
Fedora Koji to https://koji.fedoraproject.org/koji/
CentOS CBS to https://cbs.centos.org/koji/

How?